Assets/LeapMotion/Core/Scripts/XR/Editor/EyeTypeEditor.cs

```diff
@@ -12,11 +12,11 @@
 
 namespace Leap.Unity {
 
-  [CustomPropertyDrawer(typeof(EyeType))]
```

Comment:
This code is still commented out, was there a TODO here?

Reply:
Nope. So I just removed the commenting-out for now. EyeType and Leap XR Camera Control is on the chopping block but it'll be removed later when it's easier to figure out how to do that :P